lib/gpu_memory_service/client/memory_manager.py

```
@@ -605,6 +637,230 @@ def reallocate_all_handles(self, tag: str = "default") -> None:
+                    base_va,
+                    scratch.aligned_size,
+                    self.device,
+                    self._granted_lock_type,
+                )
+            except Exception:
+                # Best-effort teardown of whatever we mapped so far.
+                for h in real_handles:
+                    try:
+                        cumem_unmap(base_va, scratch.aligned_size)
+                    except Exception:
+                        pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)